pkg/frontend/admin_openshiftcluster_upgrade.go

```diff
@@ -30,6 +30,8 @@ func (f *frontend) postAdminOpenShiftUpgrade(w http.ResponseWriter, r *http.Requ
 func (f *frontend) _postAdminOpenShiftUpgrade(ctx context.Context, r *http.Request, log *logrus.Entry) error {
 	vars := mux.Vars(r)
 
+	upgradeY := r.URL.Query().Get("upgradeY") == "true"
```

Comment:
How does this works if I pass `True` or `TRUE`?

Reply:
On the Geneva side, it'll always pass `true`, but I can make this ignore case here

Reply:
how does it is configured on geneva side? checkbox? if so should be fine

Reply:
It will be a ☑, assuming everything works as intended.

Reply:
Did you tested this using geneva action code and saw?

Reply:
I will do that today

Reply:
@mjudeikis I've verified that this boolean gets set correctly by running the GA and RP locally.